experiment_scripts/rosenbrock.py

```diff
@@ -1,57 +1,188 @@
-import os
+import time
 
+import matplotlib.pyplot as plt
 import torch
-from botorch.acquisition import qKnowledgeGradient
+from botorch.acquisition import (
+    qKnowledgeGradient,
+    HybridKnowledgeGradient,
+    DiscreteKnowledgeGradient,
+    ContinuousKnowledgeGradient,
+)
 from botorch.fit import fit_gpytorch_model
+from botorch.generation import get_best_candidates, gen_candidates_torch
 from botorch.models import SingleTaskGP
+from botorch.optim import gen_batch_initial_conditions
 from botorch.optim import optimize_acqf
 from botorch.test_functions import Rosenbrock
 from botorch.utils import standardize
 from botorch.utils.sampling import manual_seed
 from botorch.utils.transforms import unnormalize
 from gpytorch.mlls import ExactMarginalLogLikelihood
 
-SMOKE_TEST = os.environ.get("SMOKE_TEST")
-
-# Defining box optimisation bounds
-bounds = torch.stack([torch.zeros(2), torch.ones(2)])
-
-# Generate XY data
-train_X = bounds[0] + (bounds[1] - bounds[0]) * torch.rand(20, 2)
 fun = Rosenbrock(dim=2, negate=True)
 fun.bounds[0, :].fill_(-5)
 fun.bounds[1, :].fill_(10)
 
+# Defining box optimisation bounds
+bounds = torch.stack([torch.zeros(2), torch.ones(2)])
```

Comment:
one line?
`bounds = torch.stack([-5 * torch.ones(2), 10 * torch.ones(2)]) `

Reply:
I could change it but this file it's just for personal use and testing. This is kinda of a free space to make a non-submittable mess XD

Reply:
done